components/server/Dockerfile

```diff
@@ -2,14 +2,18 @@ FROM python:3.10.2-buster AS compile-image
 
 RUN useradd --create-home server
 WORKDIR /home/server
-USER server
 
 RUN python -m venv venv
 ENV PATH="/home/server/venv/bin:$PATH"
 RUN python -m pip install --upgrade pip
 RUN python -m pip install wheel
-COPY requirements.txt /requirements.txt
+COPY server/requirements.txt /requirements.txt
+COPY server/requirements-internal.txt /requirements-internal.txt
+COPY shared_python /home/shared_python/
```

Comment:
Is this step really necessary? Nothing is done with this /home/shared_python in the compile-image stage. And in the final stage, the code is copied again.

Reply:
It's necessary to do the pip install the requirements-internal.txt below on line 14. Maybe it's a better solution to keep the requirements-internal out of the first stage completely.

Reply:
Actually, on closer inspection, I think the best is to leave it like this for now. In the future we should look for a solution to not install the internal dependencies with the -e argument. When that is achieved, we don't need to copy the shared_python code in the final stage anymore.